cfme/configure/settings.py

```diff
@@ -349,12 +349,20 @@ def __init__(self, appliance=None):
         Navigatable.__init__(self, appliance=appliance)
 
     @classmethod
-    def set_default_view(cls, button_group_name, default):
-        bg = ButtonGroup(button_group_name)
-        navigate_to(cls, 'All')
-        if bg.active != default:
-            bg.choose(default)
-            sel.click(form_buttons.save)
+    def set_default_view(cls, button_group_names, defaults):
+
```

Comment:
I like this enhancement, doesn't impact existing callers and allows for setting a list of views to a single given default. cool!